xmrstak/cli/cli-miner.cpp

```diff
@@ -440,6 +440,13 @@ int main(int argc, char *argv[])
 		}
 		else if(opName.compare("-u") == 0 || opName.compare("--user") == 0)
 		{
+			if(params::inst().poolURL.empty())
```

Comment:
This is not working because the order is not guaranteed.

This is creates an error: `./xmr-stak -c x -u x --url xmr.crypto-pool.fr:333`
This will not create an error `./xmr-stak -c x --url xmr.crypto-pool.fr:333 -u w`

This and the next check must be moved to [this](https://github.com/fireice-uk/xmr-stak/pull/391/files#diff-e6399cdf5a15d37895e060b979c01e31R516) function